lib/puma/launcher.rb

```diff
@@ -86,6 +86,7 @@ def initialize(conf, launcher_args={})
       else
         @runner = Single.new(self, @events)
       end
+      Puma.stats_object = @runner
```

Comment:
Nevermind - requiring 'rack/handler/puma' doesn't trigger a require of 'puma'  so `Puma.stats_object=` wasn't being defined.

Reply:
Is there an issue that needs to be fixed? Are we not testing something with the handler that we should?

Reply:
@schneems Not really sure -- Capybara was just requiring 'rack/handler/puma' when setting up puma, which used to be fine, but with this change resulted in an error because `Puma.stats_object=` wasn't defined.  If Capybara requires 'puma' as well the error goes away.  Whether or not just requiring `rack/handler/puma` should be enough is up to you.

Reply:
This should fix it https://github.com/puma/puma/pull/1546